README.rst

```diff
@@ -322,7 +322,7 @@ line. But if you prefer to call it from your code, here's how to do it:
     # Collect Args the Build Configs #
     ##################################
 
-    metabase_config = MetabaseConfig(
+    metabase_config = Metabase(
```

Comment:
What's the thinking behind this? Config suffix described the intent of the object well.

Reply:
Yeah I extended the class to be more generic and integrated the clients/parsers directly into the config. So because the classes were multipurpose, config and interface, I renamed to a more generic name. 
BUT what would be the best possible go at it is an interface class which inherits from the DbtConfig / MetabaseConfig
So I can definitely make that change. That way we keep the configs as base classes and extend them to the interface.

Reply:
I think you forgot to reinstate this change in the README.